cmd/datasource-syncer/README.md

```diff
@@ -0,0 +1,63 @@
+# Datasource Syncer
+
+This CLI tool can act as a cron job remotely syncing the given Grafana Prometheus datasource, so it has e.g.:
```

Comment:
This CLI tool acts as a cron job which remotely syncs data to a given Grafana Prometheus datasource. This ensures that the Grafana datasource has the following set correctly: